extensions/wikia/Recirculation/styles/discussions.scss

```diff
@@ -0,0 +1,181 @@
+@import 'skins/shared/mixins/flexbox';
+@import 'skins/shared/color';
+
+// DISCUSSIONS
+$discussion-height: 348px;
```

Comment:
Isn't this variable to generic? Shouldn't it be more `recirculation-discussion` or something like that?